.gitignore

```diff
@@ -20,6 +20,8 @@ test-results*.xml
 spec-xunit-reporter-0.0.1.tgz
 *xunit_*.xml
 lerna-debug.log
+apps/full-site-editing/package-lock.json
```

Comment:
I can add this in a separate PR, but we should either commit the package-lock.json or ignore it.